### PR TITLE
(feat) Remove demo-server 

### PR DESCRIPTION
**Pull request by GuiLeme**
## What does this PR do

It removes the demo-server's references from the bbb-install script and also makes Greenlight installed by default, configured to serve in the old demo-server's endpoint, the `/`.

## More

PR related to [ #15216](https://github.com/bigbluebutton/bigbluebutton/pull/15216) from bigbluebutton repository

### CODE REVIEW

File: bbb-install-2.6.sh [hunk 1]
```diff
@@ -155,9 +155,6 @@ main() {
       l)
         LETS_ENCRYPT_ONLY=true
         ;;
-      g)
-        GREENLIGHT=true
```
Comment (antobinary) [inline bbb-install-2.6.sh:159]: Did you mean to remove the other block below instead? The API_DEMOS=true

I believe we still want to keep GreenLight installation optional. The fact that we remove `-a` does not mean we should default to `-g`

Likely we want some kind of log or even preventing progress if `-a` was passed? What do you think?

Comment (GuiLeme) [reply bbb-install-2.6.sh:159]: I will return the `-g` option and remove the `-a` solely.

> Likely we want some kind of log or even preventing progress if -a was passed? What do you think?

I think logging a warning would be nice, with the message: "Installing Demo-API is not possible anymore". Does that sound good?

File: bbb-install-2.6.sh [hunk 2]
```diff
@@ -764,7 +763,7 @@ server {
 
   access_log  /var/log/nginx/bigbluebutton.access.log;
 
-  # BigBlueButton landing page.
+  # BigBlueButton asstes and static content.
```
Comment (antobinary) [inline bbb-install-2.6.sh:766]:
```suggestion
  # BigBlueButton assets and static content.
```

Comment (antobinary) [reply bbb-install-2.6.sh:766]: one more spot on like ~820

**Pull request merged**